web/src/containers/ApdStateKeyPersonnel.js

```diff
@@ -270,3 +270,10 @@ const mapDispatchToProps = {
 export default connect(mapStateToProps, mapDispatchToProps)(
   ApdStateKeyPersonnel
 );
+
+export {
+  ApdStateKeyPersonnel as plain,
+  mapStateToProps,
+  mapDispatchToProps,
+  PersonForm
+};
```

Comment:
This file doesn't have 100% coverage because it'll be impacted by #1078.  It seemed silly to spend time adding tests for code that's about to disappear.